### DISCUSSION TITLE
some suggestions for the v0.2.0 release

### DISCUSSION THREAD

**Discussion by ChristophKin** [abizovnuralem/go2_omniverse#34] [Ideas]
- in the last line of the run_sim.sh file the robot name should be --robot go2, I suppose?
- the config file for the Unitree L1 LiDAR must be in the path _IsaacLab-0.3.1/source/data/sensors/lidar/Unitree_L1.json_ or in _IsaacLab-0.3.1/source/exts/omni.isaac.sensor/data/lidar_configs/Unitree_L1.json_ and not in _IsaacLab-0.3.1/source/data/sensors/lidar/**Unitree**/Unitree_L1.json_
- the material files (AsphaltStandard.json, AsphaltWeathered.json, ...) must be copied to the _IsaacLab-0.3.1/source/data/material_files_ folder. Otherwise there are lots of `failed to load Material File` errors (I don't know if this is only a problem with my setup?)

**Comment by abizovnuralem** [comment]
@ChristophKin Thanks for your work! If it is okay, can you make a PR with these comments, please? I will be happy to approve it
